instrumentation/http/lib/opentelemetry/instrumentation/http/patches/client.rb

```diff
@@ -41,6 +46,15 @@ def annotate_span_with_response!(span, response)
             span.status = OpenTelemetry::Trace::Status.error unless (100..399).include?(status_code.to_i)
           end
 
+          def create_request_span_name(request_method, request_path)
```

Comment:
This should probably have some error handling and fall back to `HTTP #{request_method}` in the case where an error occurs.